### PR TITLE
Auto backup

### PR DESCRIPTION
**Pull request by HGinnerup**
Added {TIMESTAMP} to config, saving the config is still on to do
Also added LatestCheckpoint for resuming

### CODE REVIEW

**Review by AnjaLumholtz — APPROVED** [review]
Beautiful

**Review by AnjaLumholtz — APPROVED** [review]

File: JANGAN/ExperimentQueue/BaseRunConfig.ini [hunk 1]
```diff
@@ -43,7 +43,7 @@ MaximumLetterCount = 4096
 TrainDatasetDir = "../../Data/RealOutput/"
 TestDatasetDir = ""
 # Path of the output images
-OutputDir = "../../Data/FakeOutput/"
+OutputDir = "../../Data/{TIMESTAMP}/FakeOutput/"
```
Comment (jamadaha) [inline JANGAN/ExperimentQueue/BaseRunConfig.ini:46]: Would it be an idea to mention what {TIMESTAMP} does somewhere inside the config?

Also why remove /baseExperiment/?

**Review by jamadaha — CHANGES_REQUESTED** [review]

File: JANGAN/ExperimentQueue/BaseRunConfig.ini [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Do not change values in here for local changes
 # Add a file called 'override-config.ini' to add what changes you need.
+# Tokens can be added in the configManager, by default `{TIMESTAMP}` is replaced with a timestamp in the format of YYYYMMDD-hhmmss
```
Comment (jamadaha) [inline JANGAN/ExperimentQueue/BaseRunConfig.ini:3]:
```suggestion
# Tokens can be added in the configHelper, by default `{TIMESTAMP}` is replaced with a timestamp in the format of YYYYMMDD-hhmmss
```

**Review by AnjaLumholtz — COMMENTED** [review]
Bootiful

**Review by AnjaLumholtz — DISMISSED** [review]
Just bootiful

File: JANGAN/ExperimentQueue/BaseRunConfig.ini (hunk 1 above)
Comment (jamadaha) [reply JANGAN/ExperimentQueue/BaseRunConfig.ini:47]:
```suggestion
OutputDir = "../../Data/DataExperiment/{TIMESTAMP}/FakeOutput/"
```

File: JANGAN/ExperimentQueue/BaseRunConfig.ini [hunk 3]
```diff
@@ -43,7 +44,10 @@ MaximumLetterCount = 4000
 TrainDatasetDir = "../../Data/RealOutput/"
 TestDatasetDir = ""
 # Path of the output images
-OutputDir = "../../Data/FakeOutput/"
+OutputDir = "../../Data/{TIMESTAMP}/FakeOutput/"
+# Path to save a copy of this config
+#	Note: End with '/' to keep same filename, or end without it to also give the file name
+ConfigCopyPath = "../../Data/{TIMESTAMP}/"
```
Comment (jamadaha) [inline JANGAN/ExperimentQueue/BaseRunConfig.ini:50]:
```suggestion
ConfigCopyPath = "../../Data/BaseExperiment/{TIMESTAMP}/"
```

File: JANGAN/ExperimentQueue/BaseRunConfig.ini [hunk 4]
```diff
@@ -55,13 +59,15 @@ RefreshUIEachXIteration = 30
 # How many images the trained generator should make
 # It generates this amount of images for each type
 NumberOfFakeImagesToOutput = 10
-CheckpointPath = "checkpoints/baseExperiment/"
+CheckpointPath = "../../Data/{TIMESTAMP}/checkpoints/baseExperiment/"
```
Comment (jamadaha) [inline JANGAN/ExperimentQueue/BaseRunConfig.ini:62]:
```suggestion
CheckpointPath = "../../Data/BaseExperiment/{TIMESTAMP}/checkpoints/"
```

File: JANGAN/ExperimentQueue/BaseRunConfig.ini [hunk 5]
```diff
@@ -55,13 +59,15 @@ RefreshUIEachXIteration = 30
...
+# Where to store the latest restorepoint to resume from
+LatestCheckpointPath = "../../Data/checkpoints/baseRunConfigRef.txt"
```
Comment (jamadaha) [inline JANGAN/ExperimentQueue/BaseRunConfig.ini:64]:
```suggestion
LatestCheckpointPath = "../../Data/BaseExperiment/checkpoints/baseRunConfigRef.txt"
```

File: JANGAN/ExperimentQueue/BaseRunConfig.ini [hunk 6]
```diff
@@ -55,13 +59,15 @@ RefreshUIEachXIteration = 30
...
 # Save the model of the CGAN pr epoch
 SaveCheckpoints = True
 # If a checkpoint exist, use it, if not just train
 UseSavedModel = False
 # Directory of log output
-LogPath = "../../Data/Logs/BaseExperiment/"
+LogPath = "../../Data/{TIMESTAMP}/Logs/BaseExperiment/"
```
Comment (jamadaha) [inline JANGAN/ExperimentQueue/BaseRunConfig.ini:70]:
```suggestion
LogPath = "../../Data/BaseExperiment/{TIMESTAMP}/Logs/"
```

**Review by jamadaha — CHANGES_REQUESTED** [review]

**Review by jamadaha — APPROVED** [review]

**Pull request merged**